src/main/java/com/google/sps/servlets/RegistrationServlet.java

```diff
@@ -46,6 +46,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String lastName = Optional.ofNullable(request.getParameter("last-name"))
             .orElseThrow(() -> new IllegalArgumentException("Must fill out last name"));
     String fullName = firstName + " " + lastName;
+    String bio = Optional.ofNullable(request.getParameter("bio")).orElse(null);
```

Comment:
Instead of null, should this be an empty string? If we want to keep it null, I don't think we need an Optional here.

Reply:
Alright, I changed it